deploy/roles/r-shiny-config/defaults/main.yml

```diff
@@ -5,3 +5,5 @@ shiny_required_users: "admin"
 shiny_auth_proxy: False
 shiny_auth_proxy_header: ""
 shiny_admin_port: 4151
+
+r_shiny_groups: ""
```

Comment:
@rioh - The rest of the variables are just "shiny_" instead of "r_shiny_", I don't have a strong preference either way, but they should probably be consistent.